### ISSUE TITLE
Dashboard crashing

### ISSUE THREAD

**Issue by carolyncole** [psu-libraries/scholarsphere-3#83]
Looks like the database is not updated with the migration: https://github.com/psu-stewardship/scholarsphere/blob/develop/app/controllers/dashboard_controller.rb#L5
still is referencing scholarsphere:

2015-03-05 15:05:02 -0500 (17547) Rendering 500 page due to exception: #<Ldp::BadRequest: The namespace prefix (scholarsphere) has not been registered> - ["/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/ldp-0.2.3/lib/ldp/client/methods.rb:107:in `block in check_for_errors'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/ldp-0.2.3/lib/ldp/client/methods.rb:105:in`tap'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/ldp-0.2.3/lib/ldp/client/methods.rb:105:in `check_for_errors'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/ldp-0.2.3/lib/ldp/client/methods.rb:53:in`get'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/caching_connection.rb:11:in `block in get'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/caching_connection.rb:75:in`block in cache_resource'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/caching_connection.rb:65:in `block in log'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/notifications.rb:159:in`block in instrument'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/notifications/instrumenter.rb:20:in `instrument'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/notifications.rb:159:in`instrument'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/caching_connection.rb:64:in `log'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/caching_connection.rb:75:in`cache_resource'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/caching_connection.rb:11:in `get'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/ldp-0.2.3/lib/ldp/client/methods.rb:112:in`block in check_for_errors'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/ldp-0.2.3/lib/ldp/client/methods.rb:105:in `tap'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/ldp-0.2.3/lib/ldp/client/methods.rb:105:in`check_for_errors'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/ldp-0.2.3/lib/ldp/client/methods.rb:24:in `head'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/ldp-0.2.3/lib/ldp/resource.rb:48:in`head'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/ldp-0.2.3/lib/ldp/resource.rb:31:in `new?'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/relation/finder_methods.rb:192:in`load_from_fedora'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/relation/finder_methods.rb:179:in `find_one'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/relation/finder_methods.rb:219:in`find_with_ids'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/relation/finder_methods.rb:60:in `find'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/relation/finder_methods.rb:94:in`exists?'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/querying.rb:3:in `exists?'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/bundler/gems/sufia-7eaeef5e197b/sufia-models/app/models/proxy_deposit_request.rb:76:in`deleted_file?'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activerecord-4.1.9/lib/active_record/relation/delegation.rb:132:in `reject'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activerecord-4.1.9/lib/active_record/relation/delegation.rb:132:in`public_send'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activerecord-4.1.9/lib/active_record/relation/delegation.rb:132:in `method_missing'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activerecord-4.1.9/lib/active_record/relation/delegation.rb:99:in`method_missing'", "/opt/heracles/deploy/scholarsphere/releases/20150228205615/app/controllers/dashboard_controller.rb:5:in `index'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_controller/metal/implicit_render.rb:4:in`send_action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/abstract_controller/base.rb:189:in `process_action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_controller/metal/rendering.rb:10:in`process_action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/abstract_controller/callbacks.rb:20:in `block in process_action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:113:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:113:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:166:in`block in halting'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:166:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:166:in`block in halting'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:166:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:166:in`block in halting'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:166:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:166:in`block in halting'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:229:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:229:in`block in halting'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:229:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:229:in`block in halting'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:166:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:166:in`block in halting'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:86:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:86:in`run_callbacks'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/abstract_controller/callbacks.rb:19:in `process_action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_controller/metal/rescue.rb:29:in`process_action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_controller/metal/instrumentation.rb:31:in `block in process_action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/notifications.rb:159:in`block in instrument'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/notifications/instrumenter.rb:20:in `instrument'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/notifications.rb:159:in`instrument'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_controller/metal/instrumentation.rb:30:in `process_action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_controller/metal/params_wrapper.rb:250:in`process_action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activerecord-4.1.9/lib/active_record/railties/controller_runtime.rb:18:in `process_action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/abstract_controller/base.rb:136:in`process'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionview-4.1.9/lib/action_view/rendering.rb:30:in `process'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_controller/metal.rb:196:in`dispatch'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_controller/metal/rack_delegation.rb:13:in `dispatch'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_controller/metal.rb:232:in`block in action'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/routing/route_set.rb:82:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/routing/route_set.rb:82:in`dispatch'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/routing/route_set.rb:50:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/journey/router.rb:73:in`block in call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/journey/router.rb:59:in `each'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/journey/router.rb:59:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/routing/route_set.rb:685:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/railties-4.1.9/lib/rails/engine.rb:514:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/railties-4.1.9/lib/rails/railtie.rb:194:in `public_send'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/railties-4.1.9/lib/rails/railtie.rb:194:in`method_missing'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/journey/router.rb:73:in `block in call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/journey/router.rb:59:in`each'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/journey/router.rb:59:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/routing/route_set.rb:685:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/rack/agent_hooks.rb:30:in`traced_call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/rack/browser_monitoring.rb:23:in`traced_call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/warden-1.2.3/lib/warden/manager.rb:35:in`block in call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/warden-1.2.3/lib/warden/manager.rb:34:in `catch'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/warden-1.2.3/lib/warden/manager.rb:34:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/etag.rb:23:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/conditionalget.rb:25:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/head.rb:11:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/middleware/params_parser.rb:27:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/middleware/flash.rb:254:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/session/abstract/id.rb:225:in`context'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/session/abstract/id.rb:220:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/middleware/cookies.rb:562:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activerecord-4.1.9/lib/active_record/query_cache.rb:36:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activerecord-4.1.9/lib/active_record/connection_adapters/abstract/connection_pool.rb:621:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/active-fedora-9.0.2/lib/active_fedora/ldp_cache.rb:26:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/callbacks.rb:82:in`run_callbacks'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/middleware/callbacks.rb:27:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/middleware/remote_ip.rb:76:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/railties-4.1.9/lib/rails/rack/logger.rb:38:in `call_app'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/railties-4.1.9/lib/rails/rack/logger.rb:20:in`block in call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/tagged_logging.rb:68:in `block in tagged'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/tagged_logging.rb:26:in`tagged'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/tagged_logging.rb:68:in `tagged'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/railties-4.1.9/lib/rails/rack/logger.rb:20:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/middleware/request_id.rb:21:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/methodoverride.rb:21:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/runtime.rb:17:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/activesupport-4.1.9/lib/active_support/cache/strategy/local_cache_middleware.rb:26:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/sendfile.rb:112:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/actionpack-4.1.9/lib/action_dispatch/middleware/ssl.rb:24:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/railties-4.1.9/lib/rails/engine.rb:514:in`call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/railties-4.1.9/lib/rails/application.rb:144:in `call'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/railties-4.1.9/lib/rails/railtie.rb:194:in`public_send'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/railties-4.1.9/lib/rails/railtie.rb:194:in `method_missing'", "/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/newrelic_rpm-3.10.0.279/lib/new_relic/agent/instrumentation/middleware_tracing.rb:67:in`call'", "/opt/heracles/deploy/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/passenger-enterprise-server-5.0.1/lib/phusion_passenger/rack/thread_handler_extension.rb:69:in `process_request'", "/opt/heracles/deploy/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/passenger-enterprise-server-5.0.1/lib/phusion_passenger/request_handler/thread_handler.rb:134:in`accept_and_process_next_request'", "/opt/heracles/deploy/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/passenger-enterprise-server-5.0.1/lib/phusion_passenger/request_handler/thread_handler.rb:95:in `main_loop'", "/opt/heracles/deploy/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/passenger-enterprise-server-5.0.1/lib/phusion_passenger/request_handler.rb:438:in`block (3 levels) in start_threads'", "/opt/heracles/deploy/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/passenger-enterprise-server-5.0.1/lib/phusion_passenger/utils.rb:95:in `block in create_thread_and_abort_on_exception'"]

**Comment by awead** [comment]
I thought that was supposed to get fixed with the rake task, or is this a new issue?

**Comment by carolyncole** [comment]
@awaed There are two issues, one that all the ProxyDeposits used to have the namespace, which is fixed by the rake task and two a bug that allows files to be transferred only once and I have files that have been transferred more than one time.

The second issue is why most people's dashboard is fine, but mine is not.

**Comment by carolyncole** [comment]
This issue will be solved by the latest sufia.
